app/src/main/java/com/hedvig/app/feature/keygear/ui/itemdetail/KeyGearItemDetailActivity.kt

```diff
@@ -59,11 +63,22 @@ class KeyGearItemDetailActivity : BaseActivity(R.layout.activity_key_gear_item_d
         valuationBinder = ValuationBinder(valuationSection as LinearLayout)
         coverageBinder = CoverageBinder(coverageSection as LinearLayout)
         receiptBinder = ReceiptBinder(receiptSection as LinearLayout, supportFragmentManager)
+        nameBinder = NameBinder(nameSection as LinearLayout, this, model)
```

Comment:
I suggest you put this in the order that the sections appear on the screen, makes it slightly easier to find when navigating the code 🙂